README.md

```diff
@@ -186,19 +186,18 @@ The hash-based manager uses the fragment identifier to store navigation state an
 import { Router } from '@dojo/routing/Router';
 import { HashHistory } from '@dojo/routing/history/HashHistory';
 
-
 const router = new Router(config, HashHistory);
 ```
 
 The history manager has `current` getter, `set(path: string)` and `prefix(path: string)` APIs. The `HashHistory` class assumes the global object is a browser `window` object, but an explicit object can be provided. The manager uses `window.location.hash` and adds an event listener for the `hashchange` event. The `current` getter returns the current path, without a # prefix.
 
 ##### State History
 
-State History is coming soon!
+The state history uses, the browsers history API, `pushState()` and `replaceState()` to add or modify history entries. The state history manager requires server-side support to work effectively.
```

Comment:
The state history uses, the browsers history API, `pushState()` and `replaceState()` to add or modify history entries -> The state history uses the browser's history API, `pushState()` and `replaceState()`, to add or modify history entries